pkg/rancher-desktop/window/index.ts

```diff
@@ -182,6 +182,46 @@ export function openMain() {
   app.dock?.show();
 }
 
+let view: Electron.BrowserView;
+
+export function openExtension(id: string) {
+  // const preloadPath = path.join(paths.resources, 'preload.js');
+  console.debug(`openExtension(${ id })`);
+
+  const window = getWindow('main') ?? undefined;
+
+  if (!window) {
+    return;
+  }
+
+  const windowSize = window?.getContentSize();
+
+  if (!view) {
+    view = new BrowserView();
+    window?.setBrowserView(view);
+
+    const x = 230;
+    const y = 55;
+
+    view.setBounds({
+      x,
+      y,
+      width:  windowSize[0] - x,
+      height: windowSize[1] - y,
+    });
+
+    view.setAutoResize({ width: true, height: true });
+  }
+
+  const url = `x-rd-extension://${ id }/ui/dashboard-tab/ui/index.html`;
+
+  view.webContents
+    .loadURL(url)
+    .catch((err) => {
+      console.error(`Can't load the dashboard URL ${ url }: `, err);
```

Comment:
TODO: Show an error page instead so the user is aware. (We can actually have the default page we load be the error page, so we can just destroy the `BrowserView` here to show it.)